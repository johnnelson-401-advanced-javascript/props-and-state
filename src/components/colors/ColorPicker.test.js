import React from 'react';
import { mount } from 'enzyme';
import ColorPicker from './ColorPicker';


describe('ColorPicker', () => {
  it('Renders a ColorPicker', () => {
    const colors = ['red', 'green', 'yellow', 'blue', 'black'];
    const selectColor = () => {
      return 'black';
    };
    const wrapper = mount(<ColorPicker 
      selectColor={selectColor}
      colors={colors}/>);
    expect(wrapper).toMatchSnapshot();
  });
});
