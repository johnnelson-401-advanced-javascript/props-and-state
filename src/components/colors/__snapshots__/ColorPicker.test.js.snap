// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ColorPicker Renders a ColorPicker 1`] = `
<ColorPicker
  colors={
    Array [
      "red",
      "green",
      "yellow",
      "blue",
      "black",
    ]
  }
  selectColor={[Function]}
>
  <button
    key="red"
    onClick={[Function]}
  >
    red
  </button>
  <button
    key="green"
    onClick={[Function]}
  >
    green
  </button>
  <button
    key="yellow"
    onClick={[Function]}
  >
    yellow
  </button>
  <button
    key="blue"
    onClick={[Function]}
  >
    blue
  </button>
  <button
    key="black"
    onClick={[Function]}
  >
    black
  </button>
</ColorPicker>
`;
